src/cmd/api/api.py

```diff
@@ -0,0 +1,53 @@
+import bcrypt
+import csv
+
+
+def signUp():
+    username = input("Nieuwe gebruikersnaam: ")
+    password = input("Nieuw wachtwoord: ")
+    with open("./users.csv", 'r') as readFile:
+        reader = csv.reader(readFile, delimiter=' ')
+        for row in reader:
+            try:
+                print(row)
+                if row[0] == username:
+                    print("Username already exists")
```

Comment:
either we use english or dutch, decide on one language to keep things consistent.